tests/integration/test_sqs.py

```diff
@@ -88,12 +89,11 @@ def test_get_queue_url_contains_request_host(self, sqs_create_queue, monkeypatch
 
         # attempt to connect through a different host and make sure the URL contains that host
         host = f"http://127.0.0.1:{config.EDGE_PORT}"
-        client = aws_stack.connect_to_service(
-            "sqs",
+        client = connect_to(
```

Comment:
Also connect_externally_to please

Reply:
Changed in 250caa25fb8336c9293dcd622b4548d151f7ad34.

This is marked for removal anyway, I'll do it in a subsequent PR.